roles/core/dhcp_server/vars/CentOS_7.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
If CentOS_7.yml is identical to RedHat_7.yml, I would remove this file to rely on `os_family`.

Reply:
Done